config/index.js

```diff
@@ -28,6 +28,19 @@ var conf = convict({
       env: 'LOG_FORMAT'
     }
   },
+  memcache: {
+    address: {
+      doc: 'Hostname/IP:Port of the memcache server',
+      default: '127.0.0.1:11211',
+      env: 'MEMCACHE_ADDRESS'
+    },
+    idle: {
+      doc: 'Idle timeout for memcached connections (milliseconds)',
+      format: Number,
+      default: 30000,
```

Comment:
I had a look at the customs server code but couldn't see that happening. What am I missing?

Fwiw, something I like about this way is how it lets you run the tests against an actual memcached instance. If instead we disable memcached for the tests, it means we're not exercising part of the code.